src/Components/Aspire.Keycloak/AssemblyInfo.cs

```diff
@@ -0,0 +1,6 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire;
+
+[assembly: LoggingCategories("Keycloak")]
```

Comment:
Where does this logging category get used? (sorry I can't find it searching)

Reply:
Removed!